pkg/rancher-components/src/components/LabeledTooltip/LabeledTooltip.vue

```diff
@@ -92,6 +92,7 @@ export default defineComponent({
         z-index: z-index(hoverOverContent);
      }
 
+    // I DON'T THINK iT NEEDS FiXING... I think we delete it... CHECK EDIT->PODS->GENERAL TAB-> PULL SECRETS ::TODO-ALEX
```

Comment:
Can we create an issue for this TODO?

Reply:
@rak-phillip since you'll be off until Tuesday and also because of the GH situation, I suggest I take care of the tooltips `box-shadow` problem and the TODO comment by [this issue](https://github.com/rancher/dashboard/issues/11763) if that's ok with you

Reply:
Sure thing. I really just make sure that the TODO gets addressed more than anything else. I'm fine if #11763 will cover it.

Reply:
added as an item to 11763